package.json

```diff
@@ -5,12 +5,11 @@
   "main": "dist/cms.js",
   "scripts": {
     "start": "npm run dev",
-    "dev": "webpack-serve webpack.dev.js",
```

Comment:
`webpack.dev.js` can be deleted now, right?

Reply:
Yes, thanks for heads up. Missed it.